frontend/pages/SoftwarePage/components/AddPackageAdvancedOptions/AddPackageAdvancedOptions.tsx

```diff
@@ -81,6 +88,26 @@ const AddSoftwareAdvancedOptions = ({
               }
             />
           )}
+          {showInstallScript && (
+            <Editor
+              wrapEnabled
+              maxLines={10}
+              name="install-script"
+              onChange={onChangeInstallScript}
+              value={installScript}
+              helpText="Shell (macOS and Linux) or PowerShell (Windows)."
+              label="Install script"
+              labelTooltip={
```

Comment:
move below to match the 2 other tooltips in modal

Reply:
Confirmed tooltips for Script and Install script (the 2 under 'Advanced options') both default to top, we must have been seeing one dynamically shift to the bottom to accommodate another UI element.